File: C/Program Mengubah Nilai Angka Menjadi Nilai Huruf.c
#include <stdio.h>

int main() {
    int nilai;

    printf("Program Mengubah Nilai Angka Menjadi Nilai Huruf\n");
    printf("visit us hobingoding.com\n\n");
    // Input Nilai
    printf("Input Nilai : ");
    scanf("%d", &nilai);

    // Ubah Nilai
    if(nilai > 84 && nilai <= 100)
        printf("Nilai Huruf : A");
    else if(nilai > 79 && nilai <= 84)
        printf("Nilai Huruf : A-");
    else if(nilai > 74 && nilai <= 79)
        printf("Nilai Huruf : B+");
    else if(nilai > 69 && nilai <= 74)
        printf("Nilai Huruf : B");
    else if(nilai > 64 && nilai <= 69)
        printf("Nilai Huruf : B-");
    else if(nilai > 59 && nilai <= 64)
        printf("Nilai Huruf : C+");
    else if(nilai > 49 && nilai <= 59)
        printf("Nilai Huruf : C");
    else if(nilai > 39 && nilai <= 49)
        printf("Nilai Huruf: D");
    else if(nilai > 0 && nilai <= 39)
        printf("Nilai Huruf : E");
    else
        printf("MAAF : Masukan Anda Salah");
    return 0;
}

File: Python/Program Mengubah Nilai Angka Menjadi Nilai Huruf.py
print("Program Mengubah Nilai Angka Menjadi Nilai Huruf")
print("visit us hobingoding.com")
print()
# Input Nilai
nilai = int(input("Input Nilai : "))

# Ubah Nilai
if(nilai > 84 and nilai <= 100):
    print("Nilai Huruf : A")
elif(nilai > 79 and nilai <= 84):
    print("Nilai Huruf : A-")
elif(nilai > 74 and nilai <= 79):
    print("Nilai Huruf : B+")
elif(nilai > 69 and nilai <= 74):
    print("Nilai Huruf : B")
elif(nilai > 64 and nilai <= 69):
    print("Nilai Huruf : B-")
elif(nilai > 59 and nilai <= 64):
    print("Nilai Huruf : C+")
elif(nilai > 49 and nilai <= 59):
    print("Nilai Huruf : C")
elif(nilai > 39 and nilai <= 49):
    print("Nilai Huruf: D")
elif(nilai > 0 and nilai <= 39):
    print("Nilai Huruf : E")
else:
    print("Maaf : Masukan Anda salah")